backends/opentsdb/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTSDB with HTTP
+
+Since the version 1.16 the Netdata has the feature to communicate with OpenTSDB using HTTP API, to enable this channel
+it is necessary to set the following options in your netdata.conf
+
+```
+[backend]
+    type = opentsdb:http
+    destination = localhost:4242
+```
+
+, in this example we are considering that OpenTSDB is running with its default port(4242).
+
+## HTTPS
+
+Netdata also supports to send the metrics using SSL/TLS, this is the preferred option, because it is the safest, but OpenTDSB
```

Comment:
supports to send the metrics -> supports sending metrics

Too many commas. Split it to shorter sentences.

Reply:
Fixed!